src/main.js

```diff
@@ -467,6 +467,8 @@ function registerListeners() {
 		macOpenFiles.push(path);
 	});
 
+	global['accessibilitySupportEnabled'] = app.accessibilitySupportEnabled;
```

Comment:
Why do we need to store in `global` ? We can always read `app.accessibilitySupportEnabled` from anywhere in the main process and it will reflect the current a11y state. I feel like the global indirection is not necessary.